src/core/src/core_logic/PatchInstaller.py

```diff
@@ -88,6 +90,15 @@ def start_installation(self, simulate=False):
 
         self.composite_logger.log("\nInstalled update count: " + str(installed_update_count) + " (including dependencies)")
 
+        endTime = self.env_layer.datetime.datetime_utcnow()
+        patchService = self.package_manager.__class__.__name__
+        machine_details = TelemetryWriter.machine_info
+
+        InstallUpdatesDetails = {'job': "Install Updates", 'startTime': str(startTime), 'endTime': str(endTime), 'installed_update_count': str(installed_update_count), 'update_run_successful': str(update_run_successful),
+                             'maintenance_window': str(maintenance_window.duration), 'maintenance_window_exceeded': str(maintenance_window_exceeded), 'patchService': patchService, 'machineDetails': machine_details}
```

Comment:
High level, this is about what is expected (naming consistency) --

        patch_installation_perf_log = {'task': "PatchInstallation", 'start_time': str(start_time), 'end_time': str(end_time), 'installed_update_count': str(installed_update_count), 'update_run_successful': str(update_run_successful),
                             'maintenance_window': str(maintenance_window.duration), 'maintenance_window_exceeded': str(maintenance_window_exceeded), 'package_manager': package_manager_name, 'machine_info': machine_info}

Reply:
fixed